AdminWebsite/AdminWebsite.UnitTests/Controllers/HearingsController/EditMultiDayHearingTests.cs

```diff
@@ -491,8 +491,7 @@ public async Task Should_forward_unhandled_error_from_bookings_api()
                     FirstName = x.FirstName,
                     LastName = x.LastName,
                     ContactEmail = x.ContactEmail,
-                    DisplayName = x.DisplayName,
-                    HearingRoleCode = x.HearingRoleName
+                    DisplayName = x.DisplayName
```

Comment:
could you revert these changes please, you shouldn't need to change this test

Reply:
done